README.md

```diff
@@ -56,18 +56,60 @@ See https://getcomposer.org/ for more information and documentation.
 
 ### Inlining Css
 
-This is how to use the `CssInliner` class:
+The most basic way to use the `CssInliner` class is to create an instance with
+the original HTML, inline the external CSS, and then get back the resulting
+HTML:
 
 ```php
-$visualHtml = \Pelago\Emogrifier\CssInliner::fromHtml($html)->inlineCss($css)->render();
+use Pelago\Emogrifier\CssInliner;
+
+…
+
+$visualHtml = CssInliner::fromHtml($html)->inlineCss($css)->render();
+```
+
+If there is no external CSS file and all CSS is located within a `<style>`
+node in the HTML, you can omit the `$css` parameter:
+
+```php
+$visualHtml = CssInliner::fromHtml($html)->inlineCss()->render();
+```
+
+If you would like to get back only the content of the BODY element instead of
+the complete HTML document, you can use the `renderBodyContent` method instead:
+
+```php
+$bodyContent = $visualHtml = CssInliner::fromHtml($html)->inlineCss()
+  ->renderBodyContent();
 ```
 
-You can also use the `DOMDocument` created by `CssInliner` to process it further:
+If you would like to modify the way inlining process with any of the
+available [options](#options), you will need to call the corresponding methods
+before inlining the CSS. The could then would look like this:
 
 ```php
-$domDocument = \Pelago\Emogrifier\CssInliner::fromHtml($html)->inlineCss($css)->getDomDocument();
-$prunedHtml = \Pelago\Emogrifier\HtmlProcessor\HtmlPruner::fromDomDocument($domDocument)
-  ->removeInvisibleNodes->render();
+$visualHtml = CssInliner::fromHtml($html)->disableStyleBlocksParsing()
+  ->inlineCss($css)->render();
+```
+
+There are also some other HTML-processing classes available
+(all of which are subclasses of `AbstractHtmlProcessor`) which you can use
+to further change the HTML after inlining the CSS.
+(For more details on the classes, please have a look at the sections below.)
+`CssInliner` and all HTML-processing classes can share the same `DOMDocument`
+instance to work on:
+
+```php
+use Pelago\Emogrifier\CssInliner;
+use Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter;
+use Pelago\Emogrifier\HtmlProcessor\HtmlPruner;
+
+…
+
+$domDocument = CssInliner::fromHtml($html)->inlineCss($css)->getDomDocument();
+HtmlPruner::fromDomDocument($domDocument)->removeInvisibleNodes;
```

Comment:
Missing `()` on method call.

Reply:
Done. (And updated to the new method name.)